src/main/java/com/iota/iri/controllers/HashesViewModel.java

```diff
@@ -6,13 +6,36 @@
 import com.iota.iri.storage.Indexable;
 import com.iota.iri.storage.Tangle;
 
+/**
+ * Base implementation of a controller interface for sets of {@link com.iota.iri.model.persistables.Hashes}.
+ */
 public interface HashesViewModel {
+
+    /**Store the {@link com.iota.iri.model.persistables.Hashes} set and {@link Hash} reference to the database*/
     boolean store(Tangle tangle) throws Exception;
+
+    /**Returns the size of the {@link com.iota.iri.model.persistables.Hashes} set*/
     int size();
+
+    /**Add a hash object to the controllers referenced {@link com.iota.iri.model.persistables.Hashes} set*/
     boolean addHash(Hash theHash);
+
+    /**Returns the {@link Hash} identifier of the {@link com.iota.iri.model.persistables.Hashes} set*/
     Indexable getIndex();
+
+    /**Returns the {@link Hash} identifiers stored in the {@link com.iota.iri.model.persistables.Hashes} set*/
     Set<Hash> getHashes();
+
+    /**
+     * Removes a referenced {@link com.iota.iri.model.persistables.Hashes} set from the database
```

Comment:
```suggestion
     * Deletes a referenced {@link com.iota.iri.model.persistables.Hashes} set from the database
```
Removes made me double check that the function is not called `remove` :)